src/Nox.Generator/Infrastructure/Persistence/DbContext.template.cs

```diff
@@ -107,6 +108,15 @@ protected override void OnModelCreating(ModelBuilder modelBuilder)
         }
     }
 
+    private void ConfigureAuditable(ModelBuilder modelBuilder)
```

Comment:
@rochar the solution we talked about yesterday doesn't work on the runtime. It is because the types in Lambda function is System.RuntimeType and not System.Type as it is expected by HasQueryFiler

`modelBuilder.ForEntitiesOfType<IEntityAuditDeleted>(builder => builder.HasQueryFilter((IEntityAuditDeleted p) => p.DeletedAtUtc == null));`

So I tried many different options but it seems to be the only one working.